### ISSUE TITLE
Improve color contrast

### ISSUE THREAD

**Issue by zenorocha** [dracula/snippetslab#1]
When using purple as the background, it's important to use a dark text color. This will drastically improve readability and accessibility.

Bad:
<img width="137" alt="CleanShot 2022-07-09 at 15 58 50@2x" src="https://user-images.githubusercontent.com/398893/178125207-1479abf8-e934-464e-8189-79c85ba8a67f.png">

Good:
<img width="60" alt="CleanShot 2022-07-09 at 15 59 58@2x" src="https://user-images.githubusercontent.com/398893/178125220-f0641f4a-f1cb-4aba-8616-21ed92543d09.png">

**Comment by danielbayley** [comment]
@zenorocha Agreed, but unfortunately there is currently no way of implementing this, as there is no separate foreground setting for selected items in the sidebar…
